WeatherApp/src/main/java/org/example/ApiQuery.java

```diff
@@ -0,0 +1,7 @@
+package main.java.org.example;
+
+public class ApiQuery {
```

Comment:
To miał być interfejs, ale teraz myślę że zrobimy z tego klasę zapieczętowaną. Na razie może zostać to co jest.

Reply:
ok